components/03-modules/minicart/Minicart.html

```diff
@@ -0,0 +1,78 @@
+<div class="minicart">
+  <alpaca-list class="minicart__list">
+    <alpaca-cart-list-item
+      v-for="{ key, item } in getImageWithKey"
+      :key="key"
+      item-tag="li"
+      :product-image="item.productImage"
+      :product-name="item.productName"
+      :product-price="item.price"
+      :product-id="item.productId"
+      :product-options="item.productOptions"
+      :quantity="item.quantity"
+      :name="`qty_${key}`"
+      href="#"
+    />
+  </alpaca-list>
+  <section class="minicart__summary">
+    <alpaca-heading
+      :level="3"
+      class="
+        heading
+        heading--first-level
+        minicart__summary-heading
+      "
+    >
+      {{ summaryTitle }}
+    </alpaca-heading>
+    <div
+      v-for="total in totals"
+      :key="total.code"
+      :class="[
+        'row row--no-margins between-xs',
+        { 'minicart__grand-total-row': total.code === 'grand_total' }
+      ]"
+    >
+      <div class="col-xs-9">
+        {{ total.title }}
+      </div>
+      <div class="col-xs-3 end-xs">
+        {{ total.value }}
+      </div>
+    </div>
+  </section>
+  <div
+    class="
+      row
+      row--no-margins
+      center-xs
+      between-md
+      minicart__actions
+    "
+  >
+    <alpaca-button
+      class="
+        col-xs-12
+        col-md-3
+        minicart__return-button
+      "
```

Comment:
if class attr is less then 80 chans please don't split
`class="col-xs-12 col-md-3 minicart__return-button"`